zingo_common_components/src/protocol/activation_heights/for_test.rs

```diff
@@ -36,7 +36,7 @@ pub fn all_height_one_nus() -> parameters::testnet::ConfiguredActivationHeights
         nu5: Some(1),
         nu6: Some(1),
         // see https://zips.z.cash/#nu6-1-candidate-zips for info on NU6.1
-        nu6_1: None,
+        nu6_1: Some(1),
```

Comment:
@dorianvp I set the nu6.1 upgrade to active in most of the tests here.

I'm curious if that effects the zebra panic you saw.